dataactbroker/app.py

```diff
@@ -103,7 +83,7 @@ def runApp():
 
 if __name__ == '__main__':
     runApp()
-    proc = multiprocessing.Process(target=checkDynamo)
+    proc = multiprocessing.Process()
```

Comment:
Can we get rid of this line (and the next 2) entirely? It looks like @nmonga91 added them as part of the "check if dynamo is running on startup" story: https://github.com/fedspendingtransparency/data-act-broker-backend/commit/5db468d38b1da008e5af6cbdf14a5c0afc1aa14c